backend/lm_backend/api/routes/products.py

```diff
@@ -11,7 +11,11 @@
 router = APIRouter()
 
 
-crud_product = GenericCRUD(Product, ProductCreateSchema, ProductUpdateSchema)
+class MagicCRUD(GenericCRUD):
```

Comment:
Why only `Product` needs `MagicCRUD`?

Reply:
This was an experiment. I forgot to put away the test tubes afterward 😓